code/modules/tips_and_tricks/jobs.dm

```diff
@@ -2,22 +2,22 @@
 /tipsAndTricks/jobs
     var/list/jobs_list       //list of jobs to which tip can be shown
     textColor = "#22458d"
-    
+
 /tipsAndTricks/jobs/captain_boss
     jobs_list = list(/datum/job/captain)
-    tipText = "As a captain you own this ship. You set the rules."
+    tipText = "As a captain, you own this ship. You set the rules."
 
 /tipsAndTricks/jobs/ironhammer_theft
     jobs_list = list(/datum/job/ihc, /datum/job/gunserg, /datum/job/inspector, /datum/job/medspec, /datum/job/ihoper)
     tipText = "As an ironhammer operative, you have broad access to chase criminals. This does not mean you can take anything you have access to. Taking things from other departments is theft!"
 
 /tipsAndTricks/jobs/guild_resale_profit
     jobs_list = list(/datum/job/merchant, /datum/job/cargo_tech)
-    tipText = "As a guild merchant, you should be buying up valueable things from scavengers and reselling them for a profit. Buy anything of value that's brought to you"
+    tipText = "As a guild merchant, you should be buying up valuable things from scavengers and reselling them for a profit. Buy anything of value that's brought to you."
 
 /tipsAndTricks/jobs/captain_leader
     jobs_list = list(/datum/job/captain)
-    tipText = "A good leader's orders are always obeyed because a good leader never gives an order that won't be obeyed."
+    tipText = "A good leader's orders are always obeyed- because a good leader never gives an order that won't be obeyed."
```

Comment:
Is the dash meant to be a comma, or are you adding emphasis to the second half of the sentence?

Reply:
Yeah, the latter.